core/mono-samples/Android/AndroidSampleApp.csproj

```diff
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="BuildApp">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>android-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.Android.Sample.Mono" Version="6.0.0-*" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <Target Name="BuildNativeLib" BeforeTargets="Publish">
+    <Exec Command="cmake -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_ROOT/build/cmake/android.toolchain.cmake -DANDROID_ABI=x86_64 -DANDROID_STL=none -DANDROID_NATIVE_API_LEVEL=21 -B monodroid -DCMAKE_BUILD_TYPE=MinSizeRel -DCMAKE_C_FLAGS=%22-s -Wno-unused-command-line-argument%22" />
+    <Exec Command="cmake --build monodroid --config MinSizeRel" />
```

Comment:
`DANDROID_ABI=x86_64` limits it for x64 only

Reply:
Soft coded the architecture to allow for other architectures to be targeted.